corehq/apps/export/templates/export/partial/export_download_progress.html

```diff
@@ -52,8 +52,19 @@
                                 <span ng-show="!isMultimediaDownload">{% trans "Collecting Data" %}</span>
                                 <span ng-show="isMultimediaDownload">{% trans "Collecting Multimedia" %}</span>
                         </button>
+                        {% if request|toggle_enabled:"EMAIL_EXPORT_WHEN_DONE_BUTTON" %}
+                        <button type="button"
+                                class="btn btn-primary btn-full-width"
+                                ng-show="!isDownloadReady"
+                                ng-init="sendEmailFlag = false"
+                                ng-disabled="sendEmailFlag"
+                                ng-click="sendEmailUponCompletion(); sendEmailFlag = true;">
+                                {% trans "Email me when this is done" %}
+                                <i class="fa fa-check" ng-show="sendEmailFlag"></i>
+                        </button>
+                        {% endif %}
                         {% if show_sync_to_dropbox %}
-                        <button button="submit"
+                        <button type="submit"
```

Comment:
Curious, why was this change necessary?

Reply:
This confused us, but is just a random change. Is `<button button="submit">` normal/valid html? I assumed it was a typo.

Reply:
The original code does look like a typo, but I had thought that the default button type was `button` and switching to `submit` could change behavior. But I just looked it up and learned the [default is submit](https://stackoverflow.com/questions/31644723/what-is-the-default-button-type), so yeah, should be a fine change.

Reply:
Ha! Thanks for digging. That's kinda funny, but makes sense. This worked for the person who wrote it but not for the reason that they thought it did.